src/toolkit/matl_buy_policy.cc

```diff
@@ -267,6 +344,17 @@ double MatlBuyPolicy::SampleRequestSize() {
   return size_dist_->sample();
 }
 
+void MatlBuyPolicy::CheckActiveDormantCumulativeTimes() {
+  if (manager()->context()->time() == next_dormant_end_) {
+    SetNextActiveTime();
+    if (use_cumulative_capacity()) { next_dormant_end_ = -1; }
+    else { 
```

Comment:
`next_active_time_` is irrelevant when using cumulative capacity, so only set it when not.
```suggestion
    if (use_cumulative_capacity()) { next_dormant_end_ = -1; }
    else { 
        SetNextActiveTime();

```